README.md

```diff
@@ -2,20 +2,37 @@
 
 Simple, foolproof standard for signing arbitrary data.
 
-*   Why not [JOSE/JWS/JWT](https://jwt.io)? JSON-specific, too complicated, too
-    easy to mess up.
-*   Why not [PASETO](https://paseto.io)? JSON-specific, too opinionated.
+## Features
+
+*   Supports arbitrary message encodings, not just JSON.
+*   Authenticates the message *and* the type to avoid confusion attacks.
+*   Avoids canonicalization to reduce attack surface.
+*   Allows any desired crypto primitives or libraries.
+
+See [Background](background.md) for more information, including design
+considerations and rationale.
 
 ## What is it?
 
-*   [Signature protocol](specification.md)
-*   [Data structure](specification.md) for storing the message and signatures
+Specifications for:
+
+*   [Protocol](protocol.md) (*required*)
+*   [Data structure](envelope.md), a.k.a. "Envelope" (*recommended*)
 *   (pending #9) Suggested crypto primitives
 
 Out of scope (for now at least):
 
 *   Key management / PKI
 
+## Why not...?
+
+*   Why not raw signatures? Too fragile.
```

Comment:
Is it worth adding...
- Why not existing/legacy TUF/in-toto signature scheme? 

... from [backround.md below](https://github.com/secure-systems-lab/signing-spec/pull/23/files#diff-7e7945d9de583afbdc74bb85a519735aa7285a8d08ee8de5f0979808a8a41d4eR19-R21)?

Reply:
Added, along with a TODO comment reminding the reader to update references to "current"/"existing".